src/bundler.ts

```diff
@@ -112,149 +114,185 @@ export class Bundler {
         String(opts.inputUrl) === opts.inputUrl ? opts.inputUrl : '';
   }
 
-
   /**
-   * Return the URL this import should point to in the given bundle.
-   *
-   * If the URL is part of the bundle, this method returns `true`.
+   * Given urls to entrypoint html documents, produce a collection of bundled
+   * documents with HTML imports, external stylesheets and external scripts
+   * inlined according to the options for this Bundler.
    *
-   * If the URL is part of another bundle, this method returns the url of that
-   * bundle.
-   *
-   * If the URL isn't part of a bundle, this method returns `false`
+   * @param {Array<string>} entrypoints The list of entrypoints that will be
+   *     analyzed for dependencies. The results of the analysis will be passed
+   *     to the `strategy`. An array of length 1 will bypass the strategy and
+   *     directly bundle the document.
+   * @param {BundleStrategy} strategy The strategy used to construct the
+   *     output bundles. See 'polymer-analyzer/lib/bundle-manifest'.
```

Comment:
That's a good point.  If someone wants to *see* they should visit src instead of lib.  Fixed comments.